packages/vite-plugin-lib/package.json

```diff
@@ -18,20 +18,18 @@
   ],
   "exports": {
     ".": {
-      "types": "./dist/index.d.ts",
       "require": "./dist/index.cjs",
```

Comment:
because there is no `type` set in package.json it should automatically know that d.ts means cjs style in this sense afaik.